include/deal.II/numerics/data_out_dof_data.templates.h

```diff
@@ -1068,8 +1068,9 @@ namespace internal
 
 
 
-    template <int dim, int spacedim, typename VectorType>
-    DataEntry<dim, spacedim, VectorType>::DataEntry(
+    template <int dim, int spacedim, typename value_type>
```

Comment:
We use `value_type` as a local typedef, and `ValueType` as the name of a template argument. Alternatively, `ScalarType` would work.